driver/driver.hpp

```diff
@@ -151,7 +151,7 @@ inline void PadBufferSize(size_t& sz, int datatype_sz)
            "pool[fp16], lrn[fp16], "
            "activ[fp16], softmax[fp16], bnorm[fp16], rnn[fp16], gemm[fp16], ctc, dropout[fp16], "
            "tensorop[fp16], reduce[fp16|fp64], layernorm[bfp16|fp16], sum[bfp16|fp16], "
-           "argmax[bfp16|fp16], groupnorm[bfp16|fp16], cat[bfp16|fp16]\n");
+           "argmax[bfp16|fp16], groupnorm[bfp16|fp16], cat[bfp16|fp16], padconstant\n");
```

Comment:
Please add fp16 and bfp16 support. I know the original one doesn't have bfp16 but it doesn't seem like a big job.

Reply:
`fp16` is pretty easy, I can probably manage it
`bfp16` is kind of a pain since in the AMD HIP implementation it’s actually `ushort` so directly casting it isn’t possible. I have to look more into it

Reply:
I’ve done it, with… a fairly dirty hack. I’m open to suggestion on this, since this is the only one I can come up with